lighthouse-core/test/lib/manifest-parser-test.js

```diff
@@ -47,6 +48,16 @@ describe('Manifest Parser', function() {
     // prefer_related_applications
   });
 
+  it('should warn on invalid manifest parser URL', function() {
+    const parsedManifest = manifestParser('{}', 'not a URL', EXAMPLE_DOC_URL);
+    assert.ok(parsedManifest.warning);
+  });
+
+  it('should warn on valid but non-HTTPS manifest parser URL', function() {
+    const parsedManifest = manifestParser('{}', EXAMPLE_MANIFEST_BLOB_URL, EXAMPLE_DOC_URL);
+    assert.ok(parsedManifest.warning);
```

Comment:
we should probably learn our lesson and do some kind of real assertion :)

Reply:
heh, alright alright